sdk/src/store.rs

```diff
@@ -388,61 +389,47 @@ impl Store {
     }
 
     /// Sign the claim and return signature.
-    pub fn sign_claim(
+    #[async_generic(async_signature(
```

Comment:
good consolidation

Reply:
@mauricefisher64  I want to keep all the signing methods under a separate feature flag so web assembly size doesn't increase  for verification only.  Right now there is a feature flag called openssl_sign and that assumes openssl.  I would like a generic feature flag called sign for signing in general. Is that, ok? or can you think of a better name?

Reply:
No sure how much that would save you since the same functions libraries that sign are the same as those needed for validation.  The other way around might yield some savings (have validation as an option).  We are working on that change now.

Reply:
Having a separate feature for verfication works too. Will leave this as is for now